terraform/stacks/umccr_data_portal/workflow/template/wts_tumor_only_prod.json

```diff
@@ -50,5 +50,5 @@
       "location": "gds://production/reference-data/dragen_wts/multiqc/2022-09-14--3.9.3/SBJ01673"
     }
   ],
-  "cl_config": "{\"sample_names_replace\":{\"PRJ220412\":\"Ref_1_Good\",\"MDX210318\":\"Ref_2_Good\",\"PRJ220466\":\"Ref_3_Good\",\"PRJ211234\":\"Ref_4_Bad\",\"PRJ220790\":\"Ref_5_Bad\"}}"
+  "cl_config": "{\"sample_names_replace\":{\"PRJ220412\":\"Ref_1_Good\",\"MDX210318\":\"Ref_2_Good\",\"PRJ220466\":\"Ref_3_Good\",\"PRJ211234\":\"Ref_4_Bad\",\"PRJ220790\":\"Ref_5_Bad\",\"L2200121_dragen_qualimap\":\"Ref_1_Good\",\"L2101521_dragen_qualimap\":\"Ref_2_Good\",\"L2200188_dragen_qualimap\":\"Ref_3_Good\",\"L2101763_dragen_qualimap\":\"Ref_4_Bad\",\"L2200311_dragen_qualimap\":\"Ref_5_Bad\",}}"
```

Comment:
See [`15b78c4` (#280)](https://github.com/umccr/infrastructure/pull/280/commits/15b78c4f3702ebff04084d8a6f772284e0f05aa1#r1107825065)

Reply:
updated in https://github.com/umccr/infrastructure/commit/3b6c25b16e359aa95e5e9ad540b0ef32709e5516